ui/components/MesheryPerformance/PerformanceCard.js

```diff
@@ -94,30 +197,38 @@ function PerformanceCard({
       {/* FRONT PART */}
       <>
         <div>
-          <Typography variant="h6" component="div">
-            {name}
-          </Typography>
+          <div style={{ display : "flex", justifyContent : "space-between" }}>
+            <Typography variant="h6" component="div">
+              {name}
+            </Typography>
+            <img src={`/static/img/load-test/${loadGenerators[0]}.svg`} height="24px" />
+          </div>
           <div className={classes.noOfResultsContainer} >
             <div >
               <Typography variant="h2" component="div" style={{ marginRight : "0.75rem", color : `${theme.palette.type === "dark" ? "#fff" : "#647881"}` }}>
-                {(results).toLocaleString('en')}
+                {(results || "0").toLocaleString('en')}
               </Typography>
               <Typography variant="body1" className={classes.resultText} component="div">
                 Results
               </Typography>
             </div>
           </div>
         </div>
-        <div className={classes.bottomPart} >
-          <div className={classes.lastRunText} >
-            <div>
-              {lastRun
-                ? (
-                  <Typography variant="caption" style={{ fontStyle : "italic", color : `${theme.palette.type === "dark" ? "rgba(255, 255, 255, 0.7)" : "#647881"}` }}>
-                  Last Run: <Moment format="LLL">{lastRun}</Moment>
-                  </Typography>
-                )
-                : null}
+        <div style={{ display : "flex", justifyContent : "space-between" }}>
+          <div className={classes.bottomPart} >
+            <Link href={`https://meshery.layer5.io/user/${profile.user_id}`} target="_blank">
```

Comment:
Cam we write `https://meshery.layer5.io` as constants in utility folder and import it over here? As it is the constant link and we can re-use in future anytime if we move it to constants file. 🤔